src/Forms/Shared/Samples/UtilityNetwork/DisplayUtilityNetworkContainer/DisplayUtilityNetworkContainer.xaml

```diff
@@ -0,0 +1,67 @@
+﻿<ContentPage x:Class="ArcGISRuntimeXamarin.Samples.DisplayUtilityNetworkContainer.DisplayUtilityNetworkContainer"
+             xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Xamarin.Forms"
+             xmlns:resources="clr-namespace:Forms.Resources">
+    <ContentPage.Resources>
+        <DataTemplate x:Key="AssociationLegendItemTemplate">
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="30" />
+                    <ColumnDefinition Width="*" />
+                </Grid.ColumnDefinitions>
+                <Image Width="25"
+                       Height="25"
+                       Source="{Binding Value}" />
+                <Label Grid.Column="1" Text="{Binding Key}" />
+            </Grid>
+        </DataTemplate>
+    </ContentPage.Resources>
+    <RelativeLayout>
+        <esriUI:MapView x:Name="MyMapView"
+                        GeoViewTapped="MyMapView_GeoViewTapped"
+                        Style="{StaticResource MapWithFormStyle}" />
+        <resources:ResponsiveFormContainer x:Name="ResponsiveFormContainer">
+            <StackLayout>
+                <Label Margin="5" Text="Utility association types" />
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="auto" />
+                        <RowDefinition Height="auto" />
+                        <RowDefinition Height="auto" />
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="auto" />
+                        <ColumnDefinition Width="*" />
+                    </Grid.ColumnDefinitions>
+                    <Label Grid.Row="0"
+                           Grid.Column="1"
+                           Text="Attachment" />
+                    <Label Grid.Row="1"
+                           Grid.Column="1"
+                           Text="Connectivity" />
+                    <Label Grid.Row="2"
+                           Grid.Column="1"
+                           Text="Containment" />
+                    <Image x:Name="AttachmentImage"
+                           Grid.Row="0"
+                           Grid.Column="0"
+                           WidthRequest="25" />
+                    <Image x:Name="ConnectivityImage"
+                           Grid.Row="1"
+                           Grid.Column="0"
+                           WidthRequest="25" />
+                    <Image x:Name="ContainmentImage"
+                           Grid.Row="2"
+                           Grid.Column="0"
```

Comment:
Since everything in column 0 has the same `WidthRequest`, can the `WidthRequest` on each item be replaced with a width specified in the `ColumnDefinition`?